### ISSUE TITLE
Game Guru MAX - request of a specific thumbnail for a saved group

### ISSUE THREAD

**Issue by marcuswilm** [TheGameCreators/GameGuruRepo#1581]
Currently when a group is saved its thumbnail shows one of the items included in that group.
Example : I assembled a few items to build a house (created with a modeler) and when all is in place in the scene, I group those items and save the group, but the thumbnail shows the floor.
![ggmax_group_floor_item](https://user-images.githubusercontent.com/94678034/142685475-d4d104a3-c07a-48e8-ba66-cc4c48cf57d1.jpg)

**Comment by rvanner** [comment]
Yes, it's not ideal, it really should only make the thumbnail of the items in the group. We will aim to make this better in the future.
